tests/test_compile_models.py

```diff
@@ -106,7 +126,9 @@ def test_compile_models(
      expected_courses,
      expected_quiz_questions,
      expected_quiz_question_contents,
-     expected_quiz_multichoice_answers) = local_expected_csvs
+     expected_quiz_multichoice_answers,
+     ib_input_instances,
+     ib_pset_problems) = local_expected_csvs
```

Comment:
You may want to name these `expected_ib_input_instances` and `expected_ib_pset_problems` for consistency.